### PR TITLE
Remove required field for author in lead

### LINKED ISSUE #874: Export: Add select all / unselect all in leads table

### PR DESCRIPTION
**Pull request by AdityaKhatri**
* Change entries viz poll time from 20s to 3s
* Have consistent labeling in metadata tab (stakeholder header was larger than its adjacent field)

Fix https://github.com/the-deep/deeper/issues/874

Also merge:
https://github.com/toggle-corp/react-store/pull/672

### CODE REVIEW

File: src/views/Export/LeadsTable/index.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+const defaultProps = {
+    leads: [],
+};
+
+export default class ExportLeadsTable extends React.PureComponent {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+    static leadKeyExtractor = d => d.id
```
Comment (samshara) [inline src/views/Export/LeadsTable/index.js]: add ; for consistency

**Review by samshara — COMMENTED** [review]
make requested changes

File: src/views/Export/LeadsTable/index.js (hunk 1 above)
Comment (AdityaKhatri) [reply src/views/Export/LeadsTable/index.js]: Okayy

File: src/views/Connector/index.js [hunk 2]
```diff
@@ -77,19 +77,30 @@ const requests = {
...
-            connectors.forEach((c) => {
-                formattedConnectors[c.id] = {
-                    id: c.id,
-                    versionId: c.versionId,
-                    source: c.source,
-                    title: c.title,
-                };
-            });
+
+            connectors
```
Comment (AdityaKhatri) [inline src/views/Connector/index.js]: Lets use list to map

File: src/views/Export/index.js [hunk 3]
```diff
@@ -427,6 +370,28 @@ export default class Export extends React.PureComponent {
...
+    handleSelectAllLeads = (selectAll) => {
+        const {
+            leads: leadsFromState = [],
+        } = this.state;
+
+        const selectedLeads = listToMap(leadsFromState, d => d.id, () => selectAll);
+
+        const leads = [];
+
+        leadsFromState.forEach((l) => {
```
Comment (tnagorra) [inline src/views/Export/index.js]: Let's use Array.map, kind sir!

**Review by tnagorra — CHANGES_REQUESTED** [review]
Need to fix some problems first!

**Review by tnagorra — APPROVED** [review]

**Review by tnagorra — CHANGES_REQUESTED** [review]
Missed dsc/asc support for number in LeadGroups and Arys (RawTable)

**Review by tnagorra — APPROVED** [review]

**Pull request merged**